src/h3lib/include/mathExtensions.h

```diff
@@ -27,6 +27,14 @@
  */
 #define MAX(a, b) (((a) > (b)) ? (a) : (b))
 
+/** Evaluates to true if a + b would overflow for int32 */
+#define SUM_INT32S_OVERFLOWS(a, b) \
```

Comment:
Nit: `SUM` and `SUB` look nearly identical so I didn't even notice they were different macros in the conditionals below at first. Maybe this one can be:

```suggestion
#define ADD_INT32S_OVERFLOWS(a, b) \
```

Reply:
That's reasonable. Renamed to ADD_...